templates/layout/region--header.html.twig

```diff
@@ -6,7 +6,9 @@
   site_header__nav_position: getThemeSetting('nav_position'),
 } %}
   {% block site_header__utility_nav %}
-    {% embed "@organisms/menu/utility-nav/yds-utility-nav.twig" %}
+    {% embed "@organisms/menu/utility-nav/yds-utility-nav.twig" with {
+      utility_nav__search,
```

Comment:
This actually isn't needed at all. Twig will automatically pass variables down to includes and embeds if the variable name matches (for better or worse). So, reverting this file back to what it was will function exactly the same.

I don't think I would include this here... I could be convinced otherwise, but my default is to revert this, and leave it as it was.

Reply:
Like we talked about on Zoom, let's remove this line, and add a comment about the fact that the variable is being "silently" passed down to the Utility Nav component.